internal/config/config.go

```diff
@@ -84,24 +84,25 @@ func (c *Config) DeepCopy() *Config {
 
 // UserConfig is the set of configurations exposed to users
 type UserConfig struct {
-	Protocol          string
-	Host              string // from databricks UI
-	Port              int    // from databricks UI
-	HTTPPath          string // from databricks UI
-	Catalog           string
-	Schema            string
-	Authenticator     auth.Authenticator
-	AccessToken       string        // from databricks UI
-	MaxRows           int           // max rows per page
-	QueryTimeout      time.Duration // Timeout passed to server for query processing
-	UserAgentEntry    string
-	Location          *time.Location
-	SessionParams     map[string]string
-	RetryWaitMin      time.Duration
-	RetryWaitMax      time.Duration
-	RetryMax          int
-	Transport         http.RoundTripper
-	UseLz4Compression bool
+	Protocol                string
+	Host                    string // from databricks UI
+	Port                    int    // from databricks UI
+	HTTPPath                string // from databricks UI
+	Catalog                 string
+	Schema                  string
+	Authenticator           auth.Authenticator
+	AccessToken             string        // from databricks UI
+	MaxRows                 int           // max rows per page
+	QueryTimeout            time.Duration // Timeout passed to server for query processing
+	UserAgentEntry          string
+	Location                *time.Location
+	SessionParams           map[string]string
+	RetryWaitMin            time.Duration
+	RetryWaitMax            time.Duration
+	RetryMax                int
+	Transport               http.RoundTripper
+	UseLz4Compression       bool
+	StagingAllowedLocalPath string
```

Comment:
Where do we use StagingAllowedLocalPath?  Also in StagingCtx this member is a slice of string while here it is a single string.

Reply:
Undone